rest/blip_api_attachment_test.go

```diff
@@ -433,7 +433,7 @@ func TestBlipAttachNameChange(t *testing.T) {
 
 	// Confirm attachment is in the bucket
 	attachmentAKey := db.MakeAttachmentKey(2, "doc", digest)
-	bucketAttachmentA, _, err := rt.Bucket().GetRaw(attachmentAKey)
+	bucketAttachmentA, _, err := rt.Bucket().DefaultDataStore().GetRaw(attachmentAKey)
```

Comment:
Again, not sure why this is DefaultDataStore(), I might just be misunderstanding something.  I think attachments should stay with other user data in the collection.